tests/policies/test_policies.py

```diff
@@ -82,7 +103,7 @@ def _test_serialize_identity(env_name, model_cfg, tmpdir):
         venv,
         n_timesteps=1000,
         deterministic_policy=True,
-        rng=np.random.RandomState(0),
+        random_state=np.random.RandomState(0),
```

Comment:
Should this be using a fixture?

Reply:
I think actually here it might sense that this be manually set, since we are also seeding the environment to a specific value, and that might have something to do with it?

Reply:
Actually here it makes sense as we want to make sure that when fixing the seed you get the same result if you run it multiple times. if we use a fixture, the state of the RNG is going to change from one run to another (we would have to pass separate rngs, but I don't think it's worth writing a hack to support this using fixtures).